sdk/metricsadvisor/azure-ai-metricsadvisor/tests/test_metrics_advisor_client_live.py

```diff
@@ -60,7 +60,7 @@ def test_list_incident_root_cause(self):
             detection_configuration_id=self.anomaly_detection_configuration_id,
             incident_id=self.incident_id,
         ))
-        assert len(results) > 0
+        assert len(results) == 0
```

Comment:
we should see at least 1 incident root cause here using 
detection_config_id = "fb5a6ed6-2b9e-4b72-8b0c-0046ead1c15c"
incident_id = "2583ff47fef9174e6d6bfe73415ad914-174d2024c00"

(those are the values the live tests use)